codalab/worker/bundle_manager.py

```diff
@@ -381,6 +381,11 @@ def _construct_run_message(self, worker, bundle):
 
         # Parse |request_string| using |to_value|, but don't exceed |max_value|.
         def parse_and_min(to_value, request_string, default_value, max_value):
+            # On user-owned workers, ignore the maximum value. Users are free to
+            # use as many resources as they wish on their own machines.
+            if worker['user_id'] != self._model.root_user_id:
```

Comment:
There are really low limits set up on worksheets.codalab.org. For example, the memory limit is 10gb. This limit basically disallows running any jobs on worksheets.codalab.org that use more than 10gb, even on a user-owned machine. That doesn't seem right. If a user wants to run a job that uses 100gb, we should allow it.